latex/CMakeLists.txt

```
@@ -0,0 +1,150 @@
+find_package(Git)
+if(GIT_FOUND)
+    execute_process(COMMAND ${GIT_EXECUTABLE} describe --all --dirty --long
+        WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
+        OUTPUT_VARIABLE git_tag_description
+        OUTPUT_STRIP_TRAILING_WHITESPACE)
+endif()
+
+latex_get_output_path(output_dir)
+configure_file(version_config.tex.in "${output_dir}/version_config.tex" @ONLY)
+configure_file(git_description.tex.in "${output_dir}/git_description.tex" @ONLY)
+
+set(headers
+    accessMode.h
+    accessorBuffer.h
+    accessorImage.h
+    accessorLocal.h
+    accessTarget.h
+    atomic.h
+    atomicoperations.h
+    buffer.h
+    commandGroup.h
+    commandGroupHandler.h
+    common-byval.h
+    common-reference.h
+    context.h
+    contextInfo.h
+    deviceEvent.h
+    device.h
+    deviceInfo.h
+    deviceSelector.h
+    deviceSelectorInterface.h
+    event.h
+    eventInfo.h
+    exception.h
+    group.h
+    hitem.h
+    id.h
+    image.h
+    item.h
+    kernelInfo.h
+    kernelWIP.h
+    multipointer.h
+    nditem.h
+    ndRange.h
+    openclcInterop.h
+    parallelFor.h
+    paramTraits.h
+    platform.h
+    platformInfo.h
+    pointer.h
+    priv.h
+    programInfo.h
+    programWIP.h
+    properties.h
+    queue.h
+    queueInfo.h
+    range.h
+    sampler.h
+    stlclasses.h
+    stream.h
+    synchronization.h
+    vec.h
+)
+list(TRANSFORM headers PREPEND "headers/")
+
+set(code
+    anatomy.cpp
+    basicparallelfor.cpp
+    basicParallelForItem.cpp
+    explicitcopy.cpp
+    largesample.cpp
+    myfunctor.cpp
+    mykernel.cpp
+    myprogram.cpp
+    myprogramlink.cpp
+    parallelforbarrier.cpp
+    parallelfor.cpp
+    parallelforworkgroup2.cpp
+    parallelforworkgroup.cpp
+    propertyExample.cpp
+    singletask.cpp
```

```suggestion
    singletask.cpp
    subbuffer.cpp
```
otherwise it fails to compile.

---

Fixed.